### ISSUE TITLE
Getting Super resolution Image

### ISSUE THREAD

**Issue by JasonWUCHI** [JasonBoy1/TextZoom#45]
I was trying to get the super resolution image in demo using the pre-trained model offered by another user
https://drive.google.com/file/d/1j-g17V5kBmqS8giWNZuHe2d_GefAepCs/view?usp=sharing

I tried to get the super resolution image by adding some code in super_resolution.py line 300. However, when I save the image, the size of the image is 64x4. Could anyone teach me how to get a super resolution image that has a bigger image size?

**Comment by kanika02** [comment]
> I was trying to get the super resolution image in demo using the pre-trained model offered by another user https://drive.google.com/file/d/1j-g17V5kBmqS8giWNZuHe2d_GefAepCs/view?usp=sharing
> 
> I tried to get the super resolution image by adding some code in super_resolution.py line 300. However, when I save the image, the size of the image is 64x4. Could anyone teach me how to get a super resolution image that has a bigger image size?

Hey, How to save the output images? Can you please help me with this? Can you provide the code for saving that

**Comment by JasonWUCHI** [comment]
you can put these code under super_resolution.py line 301
# tensor to ndarray
img_save = img_sr[0]*255
img_save = np.transpose(img_save.cpu().detach().numpy() , (1,2,0))
img_save = img_save[: , : , 0:3]
img_save = np.array(img_save, dtype=np.uint8)

if np.ndim(img_save)>3:
    assert img_save.shape[0] == 1
    img_save = img_save[0]
  
img_save = Image.fromarray(img_save , "RGB")
img_save.save('demo_folder/output.png')

I made a small experiment on the model provided by the author here. I simply reorganize what JasonBoy done is this repo.
https://github.com/JasonWUCHI/TSRN-SR-Experiment-on-Computer-Text
Feel free to take a look

BTW I 100% respect the author, so if the author JasonBoy think what I did was inappropriate please let me know. I'll just remove my repo. (Though I am also Jason, I am not JasonBoy the author of this repo)
